baseframe/forms/auto.py

```diff
@@ -17,25 +18,24 @@ class ConfirmDeleteForm(Form):
     cancel = SubmitField(__(u"Cancel"))
 
 
-def render_form(form, title, message='', formid='form', submit=__(u"Submit"), cancel_url=None, ajax=False):
+def render_form(form, title, message='', formid='form', submit=__(u"Submit"), cancel_url=None, ajax=False, with_chrome=True):
     multipart = False
+    ref_id = 'form-' + buid()
     for field in form:
         if isinstance(field.widget, wtforms.widgets.FileInput):
             multipart = True
-    if form.errors:
-        code = 200  # 400
-    else:
-        code = 200
+    if not with_chrome:
+        template = THEME_FILES[current_app.config['theme']]['ajaxform.html.jinja2']
+        return render_template(template, form=form, title=title,
+            message=message, formid=formid, ref_id=ref_id, submit=submit,
+            cancel_url=cancel_url, ajax=ajax, multipart=multipart)
```

Comment:
What's the difference between this render and the one below?

Reply:
Here we are passing just the form html to the client app to render.